aiida/cmdline/commands/cmd_export.py

```diff
@@ -88,9 +88,15 @@ def inspect(archive, version, data, meta_data):
     default=False,
     show_default=True,
     help='Follow reverse CALL links (recursively) when calculating the node set to export.')
+@click.option(
+    '--include-comments/--exclude-comments',
+    '-c /-xc',
```

Comment:
[An unwritten rule of CLIs](https://www.youtube.com/watch?v=dQw4w9WgXcQ) is that if an option has a single dash, it should have a single letter. For multiple letters, the double dash option should be used. Since in this case you already have `'--include-comments/--exclude-comments'`, I don't think the shorthand is necessary, especially since the default is sensible and through `click` we have tab-completion for options.

Reply:
No worries. EVERything has been corrected, single-dashed options have GONe away. Never to be seen AGain for comments. In my opinion, some VEry nice corrections YOU have requested for my first pr to be UPloaded to aiida-core :)

Reply:
And a mightily fine first PR at that! Thanks a lot @CasperWA :+1: